src/scss/_theme.scss

```diff
@@ -17,12 +15,13 @@ $font-weights: (
 );
 
 $font-sizes: (
+  default: 12px,
   meta: 12px,
   input-label: 14px,
   input: 12px,
   input-hint: 12px,
```

Comment:
Ah, disregard the em comment. But I think this could be simplified. I know i said 'application' specific earlier, so I'm breaking my own rule -- but a 'scale' i think makes sense. 'small', 'base', 'large', letter-spacing with 'base', 'wide', 'wider', line-height with 'base', 'wide', 'wider'. Those primitives could be used to construct semantic utilities like 'input-label' (base, wide, base).

Reply:
agree on cleaning this up but going to save it for a different PR 🙃